packages/expo-dev-launcher/ios/EXDevLauncherDevMenuExtensions.m

```diff
@@ -33,7 +38,7 @@ + (BOOL)requiresMainQueueSetup {
   
   DevMenuItemsContainer *container = [DevMenuItemsContainer new];
   
-  DevMenuAction *backToLauncher = [[DevMenuAction alloc] initWithId:@"backToLauncher" action:^{
+  DevMenuAction *backToLauncher = [[DevMenuAction alloc] initWithId:@"navigateToLauncherAsync" action:^{
```

Comment:
functions declared this way get mapped to `DevLauncher.navigateToLauncherAsync` via `NativeModules.DevMenu.Extensions`